docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     privileged: true
     build: .
     ports: ["5000:5000"]
+    environment:
+      - LOGGING_LEVEL=10
```

Comment:
it's not gonna spew out tons of logs or is it a reasonable amount?

Reply:
`10` enables debug. In my opinion `20` (info) would be a better default.

Reply:
Changed to 10

Reply:
you mean 20